src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -0,0 +1,185 @@
+                width: 100%;
+                padding-bottom: 54px;
+            }
+            
+            .comments-box h3 {
+                color: var(--color-grey-dark);
+                margin: 50px;
+                font-size: 16px;
+                line-height: 43px;
+            }
+            
+            .comments-box .inputcomment {
+                @apply --layout-horizontal;
+                @apply --layout-center;
+                width: 80%;
+                margin: 32px;
+            }
+            
+            .comment-avatar {
+                @apply --layout-flex-1;
+                margin-left: 16px;
+                margin-right: 16px;
+            }
+            
+            .comments-box .inputcomment .commentbox {
+                @apply --layout-horizontal;
+                @apply --layout-center;
+                border-radius: 3px;
+                border: 1px solid var(--color-grey-lighter);
+                height: 45px;
+                width: 90%;
+                padding: 8px;
+                background-color: var(--color-grey-lightest);
+            }
+            
+            .comments-box .inputcomment .commentbox input {
+                width: 80%;
+                height: 32px;
+                font-size: 14px;
+                border: none;
+                margin: 8px;
+            }
+            
+            .comments-box .line {
+                height: 1px;
+                width: 90%;
+                display: block;
+                background-color: var(--color-white);
+                opacity: 0.2;
+            }
+            
+            .comment {
+                width: 80%;
+                height: 100px;
+                @apply --layout-horizontal;
+                @apply --layout-center;
+            }
+            
+            .content {
+                font-size: 16px;
+                font-family: var(--font-body);
+                color: var(--color-grey-dark);
+                width: 100%;
+            }
+            
+            .content p b {
+                margin-right: 6px;
+            }
+            
+            .content p {
+                margin-top: 6px;
+                margin-bottom: 6px;
+            }
+            
+            .content date-util {
+                font-size: 14px;
+                font-family: var(--font-body);
+                color: var(--color-grey-dark);
+            }
+            
+            .comments-box p {
+                font-family: var(--font-body);
+                color: var(--color-grey-dark);
+                font-size: 16px;
+            }
+            
+            iron-scroll-threshold {
+                @apply --layout-flex;
+                @apply --layout-vertical;
+            }
+            
+            span.bold {
+                font-weight: bold
+            }
+        </style>
+        <div class="comments-box">
+            <h3>Be the first to comment!</h3>
```

Comment:
Correct me if I'm wrong, but be the first to comment doesn't need to be displayed if someone already commented